### ISSUE TITLE
Build failure with aeson-2.0.x.y

### ISSUE THREAD

**Issue by phadej** [tweag/HaskellR#367]
```
src/Language/R/Debug.hs:66:11: error:
    • Couldn't match expected type ‘Key’ with actual type ‘T.Text’
    • In the first argument of ‘(.=)’, namely ‘tp’
      In the expression: tp .= go x
      In the first argument of ‘object’, namely
        ‘["header" .= info,
          "attributes"
            .= if R.unsexp x == R.unsexp attr then "loop" else toJSON attr,
          tp .= go x]’
   |
66 |         , tp .= go x
   |           ^^
```

**Comment by phadej** [comment]
ping

**Comment by facundominguez** [comment]
Thanks @phadej!

I tried a fix in  #368.

**Comment by facundominguez** [comment]
I published a new revision to `inline-r-0.10.4` with the bound `aeson < 2`. And released `inline-r-0.10.5` with the fix in #368.
